App.js

```diff
@@ -1,57 +1,36 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- * @flow
- */
-
+import { Provider } from 'react-redux';
 import React, { Component } from 'react';
-import {
-  Platform,
-  StyleSheet,
-  Text,
-  View
-} from 'react-native';
+import {View} from 'react-native'
+import configureStore from './src/config/configureStore';
+import { Navigation } from 'react-native-navigation';
+import { registerScreens } from './src/screens/screens';
+
+const store = configureStore();
+registerScreens(store, Provider);
 
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' +
-    'Cmd+D or shake for dev menu',
-  android: 'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
+const navigatorStyle = {
```

Comment:
Shouldn't they be in an separate stylesheet or smth like that?

Reply:
Okay, I will revise.